tests/unit/AccountAuthorizationDialog.spec.ts

```diff
@@ -45,8 +45,7 @@ describe('Account Authorization Dialog', () => {
     // verify and click Exit button
     const exitButton = wrapper.find('#dialog-exit-button')
     expect(exitButton.text()).toBe('Exit')
-    exitButton.trigger('click')
-    await Vue.nextTick()
+    await exitButton.trigger('click')
```

Comment:
`trigger()` and `setValue()` are async so they can be awaited, which simplifies the tests.